### PR TITLE
address page: order button

### PR DESCRIPTION
**Pull request by vladimir-8**
Fixes:

- https://github.com/coopcycle/coopcycle-web/issues/4214
- https://github.com/coopcycle/coopcycle-web/issues/4140

### CODE REVIEW

File: assets/css/styles.scss [hunk 1]
```diff
@@ -249,11 +249,11 @@ h1, h2, h3, h4, h5, h6 {
   border-bottom: 1px solid darken(#ededed, 10%);
 }
 
-.btn-payment > .fa-spinner {
+.btn > .fa-spinner {
```
Comment (Atala) [inline assets/css/styles.scss:252]: if i am not mistaken this break the spinner in the cart button when you add products on the restaurant page (the button is empty)

File: js/app/checkout/summary.js [hunk 2]
```diff
@@ -23,10 +24,7 @@ var im = new Inputmask("currency", {
     suffix: currency.position === 'postfix' ? `${space}${getCurrencySymbol()}` : ''
 })
 
-const submitBtns = document.querySelectorAll(
-  'form[name="checkout_address"] input[type="submit"],button[type="submit"]'
-)
-const mainSubmitBtn = _.find(Array.from(submitBtns), btn => !btn.hasAttribute('name'))
+const submitPageBtn = document.querySelector('.btn-submit-page')
```
Comment (Atala) [inline js/app/checkout/summary.js:27]: :+1:

**Review by Atala — CHANGES_REQUESTED** [review]
hello

the desired behavior is well implemented but the CSS change broke the spinner in js/app/restaurant/components/Order/CartButton.js and js/app/restaurant/components/Order/OrderState.js (and maybe other places, I just noticed this one by chance)

the "loading state" on the mobile web cart button seems to work fine

please have a look and then merge!

File: assets/css/styles.scss (hunk 1 above)
Comment (vladimir-8) [reply assets/css/styles.scss:252]: wow, nice catch. I'll fix it

**Comment by vladimir-8** [comment]
@Atala I fixed the issue with the loading indicator caused by my changes, and one more issue that already existed and the loading indicator had not been shown when the `Order` button was clicked on the restaurant page

**Pull request merged**